build-tools/scripts/BuildEverything.mk

```diff
@@ -55,13 +55,9 @@ ALL_AOT_ABIS = \
 	win-armeabi-v7a \
 	win-arm64 \
 	win-x86 \
-	win-x86_64
-
-ifneq ($(OS_NAME),Linux)
```

Comment:
I don't think we want this change as part of this PR, do we?

Reply:
Removed from the current version